data/publiccloud/terraform/sap/azure.tfvars

```diff
@@ -1,42 +1,79 @@
-# Launch SLES-HAE of SLES4SAP cluster nodes
+# No bastion connection
+bastion_enabled = false
 
-# Instance type to use for the cluster nodes
-hana_vm_size = "%MACHINE_TYPE%"
+# Enable all some pre deployment steps (disabled by default)
+pre_deployment = true
 
-# Disk type for HANA
-hana_data_disk_type = "Premium_LRS"
+# Region where to deploy the configuration
+az_region = "%REGION%"
+
+# Use an already existing resource group
+#resource_group_name = "my-resource-group"
+
+# Use an already existing virtual network
+#vnet_name = "my-vnet"
+
+# Use an already existing subnet in this virtual network
+#subnet_name = "my-subnet"
 
-# Caching used for HANA disk
-hana_data_disk_caching = "ReadWrite"
+# vnet address range in CIDR notation
+# Only used if the vnet is created by terraform or the user doesn't have read permissions in this
+# resource. To use the current vnet address range set the value to an empty string
+# To define custom ranges
+#vnet_address_range = "10.74.0.0/16"
+#subnet_address_range = "10.74.1.0/24"
+# Or to use already existing address ranges
+#vnet_address_range = ""
+#subnet_address_range = ""
+
+# HANA configuration (this example shows the demo option values.Find more options in the README file)
+# VM size to use for the cluster nodes
+#hana_vm_size = "Standard_E4s_v3"
 
 # Number of nodes in the cluster
 hana_count = "2"
 
-# Region where to deploy the configuration
-az_region = "%REGION%"
-
-# Installation type
-# Value can be all, skip-hana, skip-cluster
-init_type = "all"
+# Instance number for the HANA database. 00 by default.
+hana_instance_number = "00"
+
+# Network options
+#hana_enable_accelerated_networking = false
+
+# Disk configuration
+#hana_data_disks_configuration = {
+#  disks_type       = "Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS,Premium_LRS"
+#  disks_size       = "512,512,512,512,64,1024"
+#  caching          = "ReadOnly,ReadOnly,ReadOnly,ReadOnly,ReadOnly,None"
+#  writeaccelerator = "false,false,false,false,false,false"
+#  luns             = "0,1,2#3#4#5"
+#  names            = "datalog#shared#usrsap#backup"
+#  lv_sizes         = "70,100#100#100#100"
+#  paths            = "/hana/data,/hana/log#/hana/shared#/usr/sap#/hana/backup"
+#}
 
 # SLES4SAP image information
 # If custom uris are enabled public information will be omitted
 # Custom sles4sap image
+#sles4sap_uri = "/path/to/your/image"
 sles4sap_uri = "https://openqa.blob.core.windows.net/sle-images/%SLE_IMAGE%"
 
 # Custom iscsi server image
-# iscsi_srv_uri = "/path/to/your/iscsi/image"
+#iscsi_srv_uri = "/path/to/your/iscsi/image"
```

Comment:
I can understand having commented variable as documentation for future use ( well I would actually prefer to move them to some kinda README in same folder and keep config file clean but it is matter of taste ) . But I think having template , so other random value and actual value is too much for sure . Can you please remove at least template ?

Reply:
Again I want to keep the default file provided by the devs (SHAP team) as close as the default one.

Reply:
ok